Fix sort corrupting graph

msbarry opened the pull request:
Fix #1732 by making `copyProperties` not copy over the node IDs from old to new edge. No tests fail if I just remove the the call to `writeEdge` entirely in that method, but I'm assuming the reverse handling needs to be in there for a reason.

karussell commented:
Oh, this looks indeed ugly. Thanks for the quick fix! 

Please add yourself as [a contributor](https://github.com/graphhopper/graphhopper/blob/master/CONTRIBUTORS.md) and [send us the CLA](https://github.com/graphhopper/graphhopper/blob/master/CONTRIBUTING.md#license-agreement).

karussell commented on core/src/test/java/com/graphhopper/util/GHUtilityTest.java:
```diff
@@ -67,6 +68,16 @@ Graph initUnsorted(Graph g) {
+        DistanceCalc calc = new DistanceCalc2D();
+        AllEdgesIterator iter = graph.getAllEdges();
+        while (iter.next()) {
+            distance += iter.fetchWayGeometry(3).calcDistance(calc);
```
Is this also possible with iter.getDistance()?

karussell replied:
This does not work then. Will look a bit deeper whats wrong here.

msbarry replied:
getDistance doesn't seem to verify this failure in the tests so kept as-is and added a comment.

karussell commented:
Oh, this looks indeed ugly. Thanks for the quick fix! 

Please add yourself as [a contributor](https://github.com/graphhopper/graphhopper/blob/master/CONTRIBUTORS.md) and [send us the CLA](https://github.com/graphhopper/graphhopper/blob/master/CONTRIBUTING.md#license-agreement).

msbarry commented:
Thanks @karussell! Sent a request for the CLA and added myself as a contributor.  I removed the entire `writeEdge` call and all of the tests still pass, but `getDistance` doesn't seem to verify this failure in the tests so kept as-is and added a comment.

karussell commented:
Thanks!
